components/browser/toolbar/src/main/res/values/attrs_browser_toolbar.xml

```diff
@@ -15,5 +15,6 @@
     <attr name="browserToolbarSuggestionForegroundColor" format="color" />
     <attr name="browserToolbarClearColor" format="color"/>
     <attr name="browserToolbarFadingEdgeSize" format="dimension" />
+    <attr name="topProgressBar" format="boolean" />
```

Comment:
I wonder if we could make this more flexible by not using a boolean but a "gravity" flag here, like in Android layouts.

- https://developer.android.com/reference/android/view/Gravity
- https://stackoverflow.com/questions/9531177/correct-format-for-androidgravity-in-attrs-xml

So that I could set this to something like: `browserToolbarProgressBarGravity=TOP/BOTTOM`